docs/book/src/tasks/installation.md

```diff
@@ -176,55 +33,32 @@ export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
 export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"
 ```
 
-```bash
-curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-azure" asset:"infrastructure-components.yaml" version:"0.3.x"}} \
-  | envsubst \
-  | kubectl create -f -
-```
+For more information about authorization, AAD, or requirements for Azure, visit the [Azure Provider Prerequisites](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/getting-started.md#prerequisites) document.
```

Comment:
Can we keep the Azure prerequesites _before_ the base64 encoding of credentials example? The user needs to create a service principal first in order to generate credentials that can be base64 encoded, otherwise the command above will fail (the environment variables it uses for ID/Secret won't be defined and it might not be obvious to a first time user how to generate those).

Reply:
Done! let me know if there are more adjustments to do on the Azure bits